npm/metrics/prometheus-values.go

```diff
@@ -25,8 +25,12 @@ func getVecValue(gaugeVecMetric *prometheus.GaugeVec, labels prometheus.Labels)
 
 // getCountValue returns the number of times a Summary metric has recorded an observation.
 // This function is slow.
-func getCountValue(summaryMetric prometheus.Summary) (int, error) {
-	dtoMetric, err := getDTOMetric(summaryMetric)
+func getCountValue(summaryMetric interface{}) (int, error) {
```

Comment:
why `interface{}`? Will this ever be of any type other than `prometheus.Collector`?

Reply:
it can be a prometheus.Observer too. This function is only used for UTs

Reply:
changed back to prometheus.Collector and introduced a new function for getting counts for vectors